pyproject.toml

```diff
@@ -114,9 +114,9 @@ dependencies = [
 ]
 
 [tool.hatch.envs.docs.scripts]
-build = "mike deploy --push dev"
-serve = "mike serve"
-release = "version=$(python -W ignore -c 'import dagfactory; print(dagfactory.__version__)') && mike deploy --push --update-aliases $version latest"
+dev = "mkdocs build && mkdocs serve" # For local development and preventing publishing
```

Comment:
Changing this to avoid publishing from local machine

Reply:
Thanks a lot, @pankajastro !